web/wp-content/themes/borealis-2021/single-research-blogs.php

```diff
@@ -9,19 +9,28 @@
  */
 
 get_header();
-$hero_image_url = get_the_post_thumbnail_url($post->ID, 'full');
+$thumbnail_id = get_post_thumbnail_id( $post->ID );
+$alt = get_post_meta($thumbnail_id, '_wp_attachment_image_alt', true); 
+
 $title = get_the_title($post->ID);
 $publication_date = get_post_meta($post->ID, 'publication_date', true);
 $authors = get_post_meta($post->ID, 'authors', true);
 $url = get_permalink($post->ID);
+
 function getbaseurl($url)
 {
 
     $domain = parse_url($url, PHP_URL_HOST);
     return $domain;
 }
 ?>
-<div aria-hidden="true" class="pt-100 mt-19 bg-cover bg-no-repeat bg-center" style="background-image: url(<?php echo isset($hero_image_url) && !empty($hero_image_url) ? esc_url_raw($hero_image_url) : get_bloginfo('stylesheet_directory') . '/src/images/heroImage.jpg'; ?>)"></div>
+<?php if (isset($thumbnail_id) && !empty($thumbnail_id)) : ?>
```

Comment:
```suggestion
<?php if (!empty($thumbnail_id)) : ?>
```
### See above comment for context.  

In general `!empty()` gives you everything you need in `PHP`.  Because `empty` is a language construct and _not_ a built in method you can never have reference errors inside an `empty` call.  The parser sees those ahead of time and they basically catch errors.  so `isset` and `empty` almost never need to partner up